docs/index.asciidoc

```diff
@@ -405,6 +410,69 @@ Cloud ID, from the Elastic Cloud web console. If set `hosts` should not be used.
 For more details, check out the
 {logstash-ref}/connecting-to-cloud.html[Logstash-to-Cloud documentation].
 
+[id="plugins-{type}s-{plugin}-data_stream"]
+===== `data_stream`
```

Comment:
these are mostly just :scissors: from the [spec](https://github.com/elastic/logstash/issues/12178) could certainly use more data-stream related chit-chat ... // cc @karenzone